packages/protocol/contracts/L1/v1/V1Verifying.sol

```diff
@@ -70,6 +74,15 @@ library V1Verifying {
                     latestL2Height += 1;
                     latestL2Hash = fc.blockHash;
                 }
+
+                state.avgProofTime = V1Utils
```

Comment:
Do we need to move this `avgProofTime` updating to `_markBlockProven`? If there is a block's proof doesn't been submitted for a long time, this `avgProofTime` updating will not happen for at most 2047 L2 block time?  then the `uncleProofDeadline` will not be a "real time value" for those provers who submit the later 2047 L2 blocks' uncle proofs?

Reply:
I changed the PR so for the first 2*MAX_BLOCKs, we use a default, then we witch to use avgProofTime.